package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -113,15 +115,17 @@ def parse(self, **kwargs):
                     other_atom = int(other_atom) - 1
                     if i < other_atom:
                         bonds.append((i, other_atom))
-
-        # Guessing time
-        masses = guessers.guess_masses(names)
+        for n in names:
+            if n.capitalize() in SYMB2Z:
+                validated_elements.append(n.capitalize())
+            else:
+               validated_elements.append('')
```

Comment:
Could you do these changes in a separate PR, please? I would like people like @micaela-matta to have a look without having to go through this very massive PR.